contributors/devel/help-wanted.md

```diff
@@ -0,0 +1,8 @@
+# Help Wanted
+
+Items marked with the `help-wanted` label need to ensure that they are:
```

Comment:
The label is actually `help wanted` now

Reply:
Please also add a bit at the end about the `/help` and `/remove-help` commands (as detailed here: https://git.k8s.io/test-infra/commands.md)

Reply:
Okay cool. Updated help-wanted.md based on your feedback.